build-aux/org.jetuml.JetUML.yaml

```diff
@@ -0,0 +1,100 @@
+]
+command: "/app/bin/launch_jetuml"
+finish-args: [
+    "--socket=x11",
+    "--share=ipc",
+    "--device=dri",
+    "--filesystem=home",
+    "--env=PATH=/app/jre/bin:/usr/bin"
+]
+cleanup-commands: []
+cleanup: []
+modules:
+  -
+    name : "openjdk"
+    buildsystem : "simple"
+    build-commands :
+      - /usr/lib/sdk/openjdk11/install.sh
+  -
+    name: "JavaFX"
+    buildsystem: "simple"
+    builddir: true
+    build-commands:
+      - mkdir -p /app/javafx
+      - cp -r ./ /app/javafx
+    sources:
+      -
+        md5: "98d5d52df541dcc423e3e3b8bb0ab90d"
+        type: "archive"
+        url: "https://download2.gluonhq.com/openjfx/11.0.2/openjfx-11.0.2_linux-x64_bin-sdk.zip"
+  -
+    name: "Jupiter"
+    buildsystem: "simple"
+    builddir: true
+    build-commands:
+      - mkdir -p /app/junit
+      - cp -r ./ /app/junit
+    sources:
+      -
+        sha256: "380bde7098b4f6448f620afcdc2e3695896a57ee3620cfaa29ad274be1d78a61"
+        type: "file"
+        url: "https://repo1.maven.org/maven2/org/junit/jupiter/junit-jupiter/5.6.3/junit-jupiter-5.6.3.jar"
+      -
+        sha256: "26a22fe54dca351da9edccbcaa3c3cdcf25862cef3d2e04aa9363029a7beaa9d"
+        type: "file"
+        url: "https://repo1.maven.org/maven2/org/junit/jupiter/junit-jupiter-api/5.6.3/junit-jupiter-api-5.6.3.jar"
+      -
+        sha256: "12dd6cf31f4d1d0209a2f121d1c90e97254efcd7650b4041619ece13250d2ccb"
+        type: "file"
+        url: "https://repo1.maven.org/maven2/org/junit/jupiter/junit-jupiter-params/5.6.3/junit-jupiter-params-5.6.3.jar"
+      -
+        sha256: "49be6439351fdb50f0b7827e555acad26504a8084b6bc4019b2e6d73bc9da91a"
+        type: "file"
+        url: "https://repo1.maven.org/maven2/org/junit/platform/junit-platform-commons/1.6.3/junit-platform-commons-1.6.3.jar"
+      -
+        sha256: "fc68f0d28633caccf3980fdf1e99628fba9c49424ee56dc685cd8b4d2a9fefde"
+        type: "file"
+        url: "https://repo1.maven.org/maven2/org/apiguardian/apiguardian-api/1.1.1/apiguardian-api-1.1.1.jar"
+      -
+        sha256: "58812de60898d976fb81ef3b62da05c6604c18fd4a249f5044282479fc286af2"
+        type: "file"
+        url: "https://repo1.maven.org/maven2/org/opentest4j/opentest4j/1.2.0/opentest4j-1.2.0.jar"
+  -
+    name: "JetUML"
+    buildsystem: "simple"
+    builddir: true
+    build-commands:
+      - mkdir -p bin/jetuml
+      - cp -r icons/** bin/jetuml/
+      - find . -name '*.java' > files.txt
+      - /usr/lib/sdk/openjdk11/bin/javac -cp test:src -p /app/javafx/lib:/app/junit --add-modules javafx.controls,javafx.swing -d bin/jetuml @files.txt
+      - cd src; cp --parent `find -name '*.css'` ../bin/jetuml
+      - cd src; cp --parent `find -name '*.properties'` ../bin/jetuml
+      - cd test; cp --parent `find -name '*.properties'` ../bin/jetuml
+      - mkdir -p /app/bin/jetuml
+      - cp -R bin/jetuml/* /app/bin/jetuml
+    sources:
+      -
+        type: "git"
+        url: "https://github.com/louib/JetUML.git"
```

Comment:
This should point to `https://github.com/prmr/JetUML.git`, but I point to the PR's branch for testing.